lib/pages/chat_list/chat_list_item.dart

```diff
@@ -119,8 +119,7 @@ class ChatListItem extends StatelessWidget {
   Widget build(BuildContext context) {
     final isMuted = room.pushRuleState != PushRuleState.notify;
     final typingText = room.getLocalizedTypingText(context);
-    final ownMessage =
-        room.lastEvent?.senderId == Matrix.of(context).client.userID;
+    final ownMessage = room.lastEvent?.senderId == Matrix.of(context).client.userID;
```

Comment:
IMO, you should change the name of variable for clear meaning. `ownLastMessage = room.lastEvent?.senderId == Matrix.of(context).client.userID;`